lib/DDG/Spice/CaniuseLists.pm

```diff
@@ -0,0 +1,25 @@
+package DDG::Spice::CaniuseLists;
+
+# ABSTRACT: Show compatibility of various features with different browsers from caniuse.com
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d'; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+spice to => 'https://raw.githubusercontent.com/Fyrd/caniuse/master/fulldata-json/data-2.0.json';
+
+triggers startend => 'browser compatibility', 'caniuse', 'can i use', 'browser support';
+
+# Handle statement
+handle remainder => sub {
+
+    return unless $_ =~ /css[2-3]?|html5|svg|canvas|js api/i;
```

Comment:
Adding this won't allow `css css3`, etc. I had thought we might have queries like `css css3 browser compatibility`, but I'm not sure now. Do you think queries like these will make sense? If not, I'll add this and change a little in the javascript too..